eZ/Publish/Core/Repository/ObjectStateService.php

```diff
@@ -499,7 +492,7 @@ public function getContentState(ContentInfo $contentInfo, APIObjectStateGroup $o
             $objectStateGroup->id
         );
 
-        return $this->buildDomainObjectStateObject($spiObjectState);
+        return $this->buildDomainObjectStateObject($spiObjectState, $objectStateGroup);
```

Comment:
Note: this is not directly related to changes, just some optimization. If 2nd argument to `buildDomainObjectStateObject` is `null`, then `ObjectStateGroup` is loaded from the persistence layer, which is redundant.